src/resources/js/views/manager.js

```diff
@@ -291,6 +306,42 @@ tribe.events.views.manager = {};
 		return false;
 	};
 
+	/**
+	 * Catches the normal browser interactions for Next and Previous pages
+	 * so that we can use the manager to load the page requested instead
+	 * of just chaning the URL.
+	 *
+	 * @since  TBD
+	 *
+	 * @param  {Event} event DOM Event related to the window popstate
+	 *
+	 * @return {boolean}     Will always return false on this one.
+	 */
+	obj.popstate = function( event ) {
+		var $container = $( obj.selectors.container );
+		var containerData = obj.getContainerData( $container );
+
+		// Flag that we are doing popstate globally.
+		obj.doingPopstate = true;
+
+		$container.trigger( 'beforePopState.tribeEvents', event );
+
+		var url = event.originalEvent.originalTarget.location.href;
```

Comment:
We shouldn't use `originalTarget`, only FF supports it and it's experimental: https://developer.mozilla.org/en-US/docs/Web/API/Event/originalTarget

Instead, we'd probably want to use the history API to be compatible with the browsers we support.